psycop/projects/AcuteSomaticAdmission/CohortDefinition/get_somatic_emergency_visits.py

```diff
@@ -0,0 +1,71 @@
+"""
+Script for obtaining and writing all somatic emergency contacts
+for all admissions from 2012-2022. Handles LPR2 to LPR3 transition, duplicates
+and short term readmissions. Bemærk that Traumecenter er ambulant så kontakter udelukkende hertil tæller ikke
+hvilket er helt ok da patienter der udskrives direkte fra Traumecenteret ikke har været syge (for så var de jo blevet indlagt).
+Jeg skal dog være opmærksom på patienter som indlægges pga selvmordsforsøg. Dem vil der være nogen af.
+"""
+import pandas as pd
+
+from psycop.common.global_utils.sql.loader import sql_load
+from psycop.common.global_utils.sql.writer import write_df_to_sql
+from psycop.projects.forced_admission_inpatient.cohort.extract_admissions_and_visits.utils.utils import (
+    concat_readmissions_for_all_patients,
+    lpr2_lpr3_overlap,
+)
+
+
+def get_contacts_to_somatic_emergency(write: bool = False) -> pd.DataFrame:
+    # Load contact data
```

Comment:
Vi har også allerede en loader til dette: `psycop.common.feature_generation.loaders.raw.load_visits.emergency_visits`. Jeg kan se at den kalder et andet view ("[FOR_akutambulantekontakter_psyk_somatik_LPR2_inkl_2021_feb2022]"). Hvis der er uoverenstemmelse mellem de to, ville det være fedt at få dem opdateret så det rigtige view bliver kaldt af emergency_visits.

Igen kan der i kaldet til emergency_visits() specificeres en shakkode så det kun er data fra somatikken du får

Reply:
Smart. Jeg har valgt at bruge en anden tabel da jeg egentlig ønsker akutte indlæggelser i somatikken. Har skrevet koden så det passer. Jeg kunne sikkert med fordel lave en loader. Men der vil jeg få brug for hjælp. Spørger næste gang jeg mødet en af jer på Børglumvej.